o-fish-ios/Views/PreboardingFlow/VesselRecordView/VesselRecordItemView.swift

```diff
@@ -22,16 +22,16 @@ struct VesselRecordItemView: View {
         let citations = report.inspection.summary.violations.filter({ $0.disposition == .citation }).count
 
         if warnings > 0 {
-            warningString = "\(warnings) \(warnings == 1 ? "Warning" : "Warnings")"
+            warningString = "\(warnings) \(warnings == 1 ? NSLocalizedString("Warning", comment: "") : NSLocalizedString("Warnings", comment: ""))"
```

Comment:
I don't believe you should use `NSLocalizedString` in SwiftUI unless you're including the comment to provide extra context for the translators?

Reply:
Wrote comments